tests/wic.rs

```diff
@@ -5,7 +5,7 @@ use windows::Win32::UI::Shell::SHCreateMemStream;
 
 #[test]
 fn basic() {
-    unsafe { CoInitialize(None) }.expect("CoInitialize");
+    unsafe { CoInitialize(None) }.unwrap();
```

Comment:
Why this change? 🤔

Reply:
CoInitialize now return a custom type that only have `unwrap` and not `expect`. But I found a method to change it to a standard `Result` type.